server/data/mappers/updateInductionMapper.ts

```diff
@@ -0,0 +1,18 @@
+import type { UpdateInductionRequest } from 'educationAndWorkPlanApiClient'
+import type { CreateOrUpdateInductionDto } from 'inductionDto'
+
+function toUpdateInductionRequest(updateInductionDto: CreateOrUpdateInductionDto): UpdateInductionRequest {
+  return {
+    reference: updateInductionDto.reference,
+    prisonId: updateInductionDto.prisonId,
+    workOnRelease: updateInductionDto.workOnRelease,
+    previousQualifications: updateInductionDto.previousQualifications,
+    previousTraining: updateInductionDto.previousTraining,
+    previousWorkExperiences: updateInductionDto.previousWorkExperiences,
+    inPrisonInterests: updateInductionDto.inPrisonInterests,
+    personalSkillsAndInterests: updateInductionDto.personalSkillsAndInterests,
+    futureWorkInterests: updateInductionDto.futureWorkInterests,
+  } as UpdateInductionRequest
```

Comment:
Do you need the cast `as UpdateInductionRequest` on this line? If you do, it might suggest you are missing some fields?

Also, given that it looks like its a direct field to field mapping, would this work (might be too concise for your liking?)

```
return { ...updateInductionDto }
```